salt/_states/metalk8s.py

```diff
@@ -76,6 +77,7 @@ def static_pod_managed(name,
             context or {},
             config_digest=config_digest, metalk8s_version=metalk8s_version
         ),
+        tmp_dir=kwargs.pop("tmp_dir", tempfile.gettempdir()),
```

Comment:
Minor, maybe deserve a small comment :smile:

Reply:
This line is going away, I'm going to push the latest code so you can have a look :)